test/unit/specs/components/ActionModal/components/TableInvoice.spec.js

```diff
@@ -6,25 +6,31 @@ describe(`TableInvoice`, () => {
   const localVue = createLocalVue()
   localVue.directive(`tooltip`, () => {})
 
-  const $store = {
-    getters: {
-      bondDenom: `STAKE`
-    }
-  }
-
   beforeEach(() => {
     wrapper = shallowMount(TableInvoice, {
       localVue,
-      mocks: { $store },
       propsData: {
-        amount: 1,
+        amount: 17.2,
         gasEstimate: 1234567,
-        gasPrice: 2.5e-8
+        gasPrice: 2.5e-8,
+        bondDenom: `STAKE`
       }
     })
   })
 
-  it(`should show the transaction invoice summary`, async () => {
+  it(`should render correctly`, async () => {
     expect(wrapper.vm.$el).toMatchSnapshot()
   })
+
+  it(`should display the correct subtotal`, async () => {
```

Comment:
you need to test the computed properties, not the rendering of the component in a specific scenario. 

eg. `networkFee` with different gas prices and gas estimates.